lingvo/tasks/asr/params/librispeech_ctc.py

```diff
@@ -125,12 +137,17 @@ def Task(self):
     tp.scale_gradients = False
     tp.l2_regularizer_weight = None
 
-    # Setting p.eval.samples_per_summary to a large value ensures that dev,
-    # devother, test, testother are evaluated completely (since num_samples for
-    # each of these sets is less than 5000), while train summaries will be
-    # computed on 5000 examples.
-    p.eval.samples_per_summary = 5000
-    p.eval.decoder_samples_per_summary = 5000
+    # A value of 0 will force each eval and dev task to iterate over
+    # the entire dataset.
+    p.eval.samples_per_summary = 0
+    p.eval.decoder_samples_per_summary = 0
+
+    # These need to be stored on google cloud and copied to local disk
+    # so that C++ code can access them via posix file system APIs
+    p.enable_wfst_decoder = True
+    p.tlg_path = "/home/ws15dgalvez/lingvo-copy/scripts/recipes/librispeech/work_1a/data/lang_char_tgsmall/TLG.fst"
```

Comment:
@anjaligopi I need to make these publicly accessible on google cloud, but also download them from google cloud to the temporary filesystem (or somewhere) so that the C++ code can load them (it doesn't know how to read from google cloud storage, only from POSIX file systems). Altenratively, for now, you can "google compute scp" these files from my instance "dt" to yours.